routes/routes.py

```diff
@@ -46,7 +47,24 @@ def compare_route():
     if None != request.args.get("rest1") and None != request.args.get("rest2"):
         rest1 = request.args.get("rest1")
         rest2 = request.args.get("rest2")
-        return RestaurantComparer().compare_restaurants(rest1=rest1, rest2=rest2)
+        return str(RestaurantComparer().compare_restaurants(rest1=rest1, rest2=rest2))
 
     else:
         return "errorMessage: One or more restaurant IDs are missing"
+
+
+# receives longitude and latitude, as well as restaurant id
+# returns list of restaurants, from most compatible to least
+@app.route("/restaurants/search_and_find", methods=["GET"])
```

Comment:
So this method takes in a resturant and its location and returns similar resturants. What is the search radius? Is it defaulted?